<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cerrar sesi√≥n</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_usuarios">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>631</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Nombres</string>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellidos</string>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Correo</string>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nacimiento</string>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_2_buscar">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>90</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_correo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_orden">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Ninguno</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Preorden</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inorden</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Postorden</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cascadia Mono SemiBold</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SocialStructure</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCarga_masiva">
    <property name="title">
     <string>Carga masiva</string>
    </property>
    <addaction name="actionCargar_usuarios"/>
    <addaction name="actionCargar_solicitudes"/>
    <addaction name="actionCargarPublicaciones"/>
   </widget>
   <widget class="QMenu" name="menuReportes">
    <property name="title">
     <string>Reportes</string>
    </property>
    <addaction name="actionUsuarios"/>
    <addaction name="actionPublicaciones"/>
    <addaction name="actionLista_de_amistades"/>
    <addaction name="actionGrafo_de_amistades"/>
   </widget>
   <widget class="QMenu" name="menuBuscar">
    <property name="title">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComprimir">
    <property name="title">
     <string>Guardar</string>
    </property>
    <addaction name="actionUsuarios_comprimir"/>
    <addaction name="actionAmigos"/>
    <addaction name="actionSolicitudes_recibidas"/>
    <addaction name="actionSolicitudes_enviadas"/>
   </widget>
   <addaction name="menuCarga_masiva"/>
   <addaction name="menuReportes"/>
   <addaction name="menuBuscar"/>
   <addaction name="menuComprimir"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCargar_usuarios">
   <property name="text">
    <string>Cargar usuarios</string>
   </property>
  </action>
  <action name="actionCargar_solicitudes">
   <property name="text">
    <string>Cargar solicitudes</string>
   </property>
  </action>
  <action name="actionCargarPublicaciones">
   <property name="text">
    <string>Cargar publicaciones</string>
   </property>
  </action>
  <action name="actionUsuarios">
   <property name="text">
    <string>Reporte usuarios</string>
   </property>
  </action>
  <action name="actionPublicaciones">
   <property name="text">
    <string>Reporte publicaciones</string>
   </property>
  </action>
  <action name="actionUsuarios_comprimir">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="actionAmigos">
   <property name="text">
    <string>Amigos</string>
   </property>
  </action>
  <action name="actionSolicitudes_recibidas">
   <property name="text">
    <string>Solicitudes recibidas</string>
   </property>
  </action>
  <action name="actionSolicitudes_enviadas">
   <property name="text">
    <string>Solicitudes enviadas</string>
   </property>
  </action>
  <action name="actionLista_de_amistades">
   <property name="text">
    <string>Lista de amistades</string>
   </property>
  </action>
  <action name="actionGrafo_de_amistades">
   <property name="text">
    <string>Grafo de amistades</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
